src/store/modules/search.store.js

```diff
@@ -129,16 +98,14 @@ const actions = {
                 dispatch('setPinnedLocation', entry.coordinates)
                 break
         }
-        commit('setSearchQuery', entry.getSimpleTitle())
-        commit('hideSearchResults')
+        dispatch('setSearchQuery', { query: entry.getSimpleTitle() })
     },
 }
 
 const mutations = {
     setSearchQuery: (state, query) => (state.query = query),
-    setSearchResults: (state, results) => (state.results = results ? results : []),
-    showSearchResults: (state) => (state.show = true),
-    hideSearchResults: (state) => (state.show = false),
+    setSearchResults: (state, results) =>
+        (state.results = results ? results : new CombinedSearchResults()),
```

Comment:
you can `state.results = results || new CombinedSearchResulsts()` if you want to remove the ternary operator